app/database/schema/user_schema.js

```diff
@@ -0,0 +1,64 @@
+//This file holds the schema of the Database
+//Check out the readme in this folder for more information
+
+const mongoose = require("mongoose");
+
+const userSchema = new mongoose.Schema({
+  userName: String,
+  userPassword: String,
```

Comment:
The password should be stored as a hash rather than a string. May be easier to open a new issue to update it

Reply:
Good looking out, unfortunately, mongoose doesn't have a hash type but npm has some packages that can hash data, I will open up a new issue for it and will assign to myself.